app/src/main/java/org/mozilla/fenix/settings/logins/fragment/EditLoginFragment.kt

```diff
@@ -231,21 +239,26 @@ class EditLoginFragment : Fragment(R.layout.fragment_edit_login) {
             inputLayoutUsername?.let {
                 usernameChanged = true
                 validUsername = false
-                it.setErrorIconDrawable(R.drawable.mozac_ic_warning)
                 it.error = context?.getString(R.string.saved_login_duplicate)
+                it.setErrorIconDrawable(R.drawable.mozac_ic_warning_with_bottom_padding)
+                it.setErrorIconTintList(ColorStateList.valueOf(ContextCompat.getColor(requireContext(), R.color.design_error)))
+                clearUsernameTextButton.isVisible = false
             }
         } else {
             usernameChanged = true
             validUsername = true
             inputLayoutUsername.error = null
+            inputLayoutUsername.errorIconDrawable = null
+            clearUsernameTextButton.isVisible = true
         }
     }
 
     private fun setPasswordError() {
         inputLayoutPassword?.let { layout ->
             validPassword = false
             layout.error = context?.getString(R.string.saved_login_password_required)
-            layout.setErrorIconDrawable(R.drawable.mozac_ic_warning)
+            layout.setErrorIconDrawable(R.drawable.mozac_ic_warning_with_bottom_padding)
+            layout.setErrorIconTintList(ColorStateList.valueOf(ContextCompat.getColor(requireContext(), R.color.design_error)))
```

Comment:
L244 and L261 are hitting the max line length error, you can autoformat with Android Studio or just manually break it into multiple lines!